packages/trader/src/App/Components/Elements/PositionsDrawer/helpers/positions-helper.ts

```diff
@@ -50,9 +58,9 @@ export const filterByContractType = (
     const is_high_low = isHighLow({ shortcode });
     const is_vanilla = isVanillaContract(contract_type);
     const trade_types = is_call_put
-        ? ['CALL', 'CALLE', 'PUT', 'PUTE']
+        ? [CONTRACT_TYPES.CALL, CONTRACT_TYPES.CALLE, CONTRACT_TYPES.PUT, CONTRACT_TYPES.PUTE]
```

Comment:
```
const { CALL, CALLE, PUT, PUTE } = CONTRACT_TYPES;
```
nit-pick: it seems to look nicer when destructured :)

```suggestion
        ? [CALL, CALLE, PUT, PUTE]
```